recipes/tabulate/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+mkdir build && cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DBUILD_SHARED_LIBS=ON \
+      -DCMAKE_FIND_ROOT_PATH=$PREFIX \
```

Comment:
you don't need most of the variables here because you don't build a (shared) library.

you can remove everything except 

```
-DCMAKE_INSTALL_PREFIX=$PREFIX \
-DCMAKE_BUILD_TYPE=Release \
```